bootstrap/bootstrap.go

```diff
@@ -431,11 +432,42 @@ func (b *Bootstrap) PluginPhase() error {
 		checkout, err := b.checkoutPlugin(p)
 		if err != nil {
 			return errors.Wrapf(err, "Failed to checkout plugin %s", p.Name())
-
+		}
+		if b.Config.PluginValidation && checkout.Definition == nil {
```

Comment:
How would `checkout.Definition` (not) be nil here? Wouldn't it always be nil?

Reply:
Fixed!